### PR TITLE
(#69) bumped Cake references to v2.0.0

### LINKED ISSUE #69: Update Cake.XdtTransform add-in to target Cake v2.0.0
[Cake 2.0.0](https://cakebuild.net/blog/2021/11/cake-v2.0.0-released) has been released 🎉

### PR DESCRIPTION
**Pull request by nils-a**
and updated .NET SDK to 6.0 in appVeyor

fixes #69

### CODE REVIEW

File: src/Cake.XdtTransform.Tests/Cake.XdtTransform.Tests.csproj [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
     <LangVersion>default</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Cake.Testing" Version="1.0.0" />
+    <PackageReference Include="Cake.Testing" Version="2.0.0" />
     <PackageReference Include="DotNet.Xdt" Version="2.2.1" />
     <PackageReference Include="FluentAssertions" Version="5.10.3" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.9.1" />
```
Comment (gitfool) [inline src/Cake.XdtTransform.Tests/Cake.XdtTransform.Tests.csproj]: Bump other dependencies too:
```suggestion
    <PackageReference Include="FluentAssertions" Version="6.2.0" />
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.0.0" />
```

File: src/Cake.XdtTransform/Cake.XdtTransform.csproj [hunk 2]
```diff
@@ -30,7 +30,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Cake.Core" Version="1.0.0" PrivateAssets="All" />
+    <PackageReference Include="Cake.Core" Version="2.0.0" PrivateAssets="All" />
     <PackageReference Include="DotNet.Xdt" Version="2.2.1" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
```
Comment (gitfool) [inline src/Cake.XdtTransform/Cake.XdtTransform.csproj]: Bump other dependencies too:
```suggestion
    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
```

File: .appveyor.yml [hunk 3]
```diff
@@ -1,7 +1,21 @@
...
+#---------------------------------#
+#  Install .NET 6                 #
+#---------------------------------#
+install:
+  - ps: $env:DOTNET_INSTALL_DIR = "$pwd\.dotnetsdk"
+  - ps: mkdir $env:DOTNET_INSTALL_DIR -Force | Out-Null
+  - ps: Invoke-WebRequest -Uri "https://dotnet.microsoft.com/download/dotnet/scripts/v1/dotnet-install.ps1" -OutFile "$($env:DOTNET_INSTALL_DIR)/dotnet-install.ps1"
+  - ps: '& "$($env:DOTNET_INSTALL_DIR)/dotnet-install.ps1" -Version 2.1.818 -InstallDir $env:DOTNET_INSTALL_DIR'
+  - ps: '& "$($env:DOTNET_INSTALL_DIR)/dotnet-install.ps1" -Version 3.1.414 -InstallDir $env:DOTNET_INSTALL_DIR'
+  - ps: '& "$($env:DOTNET_INSTALL_DIR)/dotnet-install.ps1" -Version 5.0.402 -InstallDir $env:DOTNET_INSTALL_DIR'
```
Comment (gitfool) [inline .appveyor.yml]:
```suggestion
  - ps: '& "$($env:DOTNET_INSTALL_DIR)/dotnet-install.ps1" -Version 3.1.415 -InstallDir $env:DOTNET_INSTALL_DIR'
  - ps: '& "$($env:DOTNET_INSTALL_DIR)/dotnet-install.ps1" -Version 5.0.403 -InstallDir $env:DOTNET_INSTALL_DIR'
```

File: src/Cake.XdtTransform.Tests/Cake.XdtTransform.Tests.csproj (hunk 1 above)
Comment (nils-a) [reply src/Cake.XdtTransform.Tests/Cake.XdtTransform.Tests.csproj]: Personally, I would rather bump the "different" dependencies in a separate PR, so the changes show up nicely in separate issues. 
However, I'm not against bumping those in the same PR, if @phillipsj is fine with that.

File: src/Cake.XdtTransform/Cake.XdtTransform.csproj (hunk 2 above)
Comment (nils-a) [reply src/Cake.XdtTransform/Cake.XdtTransform.csproj]: See my comment in https://github.com/phillipsj/Cake.XdtTransform/pull/70#discussion_r767144437

File: .appveyor.yml (hunk 3 above)
Comment (nils-a) [reply .appveyor.yml]: yes, you're right. 2.1 is really not needed. Good catch. 👍

File: src/Cake.XdtTransform.Tests/Cake.XdtTransform.Tests.csproj (hunk 1 above)
Comment (phillipsj) [reply src/Cake.XdtTransform.Tests/Cake.XdtTransform.Tests.csproj]: Sure bumping those in the same PR is fine with me.

Comment (gitfool) [reply src/Cake.XdtTransform.Tests/Cake.XdtTransform.Tests.csproj]: @nils-a sure, no problem. I just used a suggestion for the convenience of expressing the change.

Comment (nils-a) [reply src/Cake.XdtTransform.Tests/Cake.XdtTransform.Tests.csproj]: done.

File: src/Cake.XdtTransform/Cake.XdtTransform.csproj (hunk 2 above)
Comment (nils-a) [reply src/Cake.XdtTransform/Cake.XdtTransform.csproj]: done.

**Comment by daghsentinel** [comment]
Hi! Any chance of getting this reviewed and made available, @phillipsj ?

**Comment by phillipsj** [comment]
I will try to get that done this week.

**Review by phillipsj — APPROVED** [review]
LGTM

**Comment by phillipsj** [comment]
@daghsentinel this has bee published to NuGet.

**Comment by daghsentinel** [comment]
Many thanks! I wish for a happy new year for you.

**Pull request merged**